sycl/source/detail/filter_selector_impl.cpp

```diff
@@ -129,6 +131,7 @@ int filter_selector_impl::operator()(const device &Dev) const {
       else
         DeviceTypeOK = (DT == Filter.DeviceType);
     }
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
```

Comment:
Are these `#ifdef` lines changing the behavior of the `filter_selector`?  I'm not sure we want to do that, and I didn't see any changes to the "sycl_ext_oneapi_filter_selector" extension specification.

Reply:
Thanks, I've fixed this. It was indeed changing the behavior of filter selector.